content/blog/2018/istio-authorization/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Micro-Segmentation with Istio Authorization
+description: Describe Istio authorization characterics and how it can be used for different use cases.
+publishdate: 2018-07-20
+subtitle:
+attribution: Limin Wang
+weight: 87
+---
+
+[Istio's authorization feature](/docs/concepts/security/#authorization), also known as Istio Role Based Access Control,
+provides access control for services in an Istio mesh. Istio authorization is Istio’s native authorization support.
+It can be used on any platform that Istio supports. It features:
+
+* Authorization at different levels of granularity, including namespace level, service level, and method level.
+* Service-to-service and end-user-to-service authorization.
+* High performance, as it is enforced natively on Envoy.
+* Role-based semantics, which makes it easy to use.
+* High flexibility as it allows users to define conditions using
+[combination of Istio attributes](/docs/reference/config/authorization/constraints-and-properties/).
+
+In this blog post, we first elaborate Istio authorization characteristics, next we show how you could use Istio
```

Comment:
Please avoid using "we". https://preliminary.istio.io/about/contribute/style-guide/#avoid-using-we. Suggest:

In this blog post, you'll learn about the main authorization features and how to use them in different situations.

Reply:
Fixed it here and a few other places.